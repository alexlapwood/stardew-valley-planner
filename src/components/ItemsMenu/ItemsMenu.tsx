import * as React from "react";

import SeedMenu from "./SeedMenu/SeedMenu";

interface IProps {
  date: number;
  selectCrop: (cropId: string) => void;
  selectedItem?: ISelectedItem;
}

const ItemsMenu: React.SFC<IProps> = props => (
  <SeedMenu
    date={props.date}
    selectCrop={props.selectCrop}
    selectedItem={props.selectedItem}
  />
);

export default ItemsMenu;
