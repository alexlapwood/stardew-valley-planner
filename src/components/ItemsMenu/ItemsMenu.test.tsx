import { shallow } from "enzyme";
import * as React from "react";

import ItemsMenu from "./ItemsMenu";

it("renders correctly", () => {
  const itemsMenu = shallow(<ItemsMenu date={0} selectCrop={jest.fn()} />);

  expect(itemsMenu).toMatchSnapshot();
});
