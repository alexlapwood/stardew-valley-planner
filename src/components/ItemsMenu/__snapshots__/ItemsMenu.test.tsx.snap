// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<SeedMenu
  date={0}
  selectCrop={[MockFunction]}
/>
`;
